data/card_catalogue/unique-1.json

```diff
@@ -144,7 +94,7 @@
     "upgrade": "",
     "tier": "green",
     "details": {
-      "rules": "All players return to Gerraro's Hideout. Return this card to the deck.",
+      "rules": "All players return to the bed they last slept in",
```

Comment:
I think this card limits the design space for levels too much. Escaping is fun and this card prevents us from using it as a mechanic

Reply:
> I think this card limits the design space for levels too much, even with the single use requirement. Escaping is fun and this card prevents us from using it as a mechanic
> 
> I think I might delete it, thoughts @elliottomlinson ?

I'd be up for keeping it so long as it's "the last place they rested" for flexibility. I'm up for keeping Gerraro's Door in as a gold as well 👍